src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -191,7 +191,13 @@ public function convert(FileVersion $fileVersion, $formatKey, $imageFormat)
 
         // Set Interlacing to plane for smaller image size.
         if (1 == count($image->layers())) {
-            $image->interlace(ImageInterface::INTERLACE_PLANE);
+            try {
+                $image->interlace(ImageInterface::INTERLACE_PLANE);
+            } catch (\InvalidArgumentException $exception) {
+                // ignore unsupported Interface type
```

Comment:
this seems have nothing todo with the [vips](https://github.com/rokka-io/imagine-vips/blob/3039f6a27d481c4a61363b68a910a8110973bcd1/lib/Imagine/Vips/Image.php#L505) adapter what case does this then happen should maybe be not part of this PR?

Reply:
not directly but thats a possible exception which is thrown in other adapters of imagine

Reply:
Which does not automatically mean that we should simply catch and ignore. This way we won't even realize that an exception happened. Or is that in any way required in order to work?

Reply:
yes - if you have installed the vips adapter (or any other adapter) which does not implement interlace sulu will crash :) we have to ignore it - as we also do it on other places in this class

Reply:
But as @alexander-schranz already described `interlace` in that case throws a `RuntimeException` (which I also don't like, but that seems to be the way it is...) not an `InvalidArgumentException`, which could also have loads of other causes.

Reply:
OK will remove the catch